packages/components/addon/template-registry.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type HdsButtonIndexComponent from '@hashicorp/design-system-components/components/hds/button';
```

Comment:
a few considerations:
- this list is going to become quite long, better to keep it organized in a similar way as other lists (eg, when we import the CSS files of the different components) 
- should we consider updating the blueprints so that much of this code is generated automatically (or at least a placeholder is added, so the developer doesn't forget to)

Reply:
I've not made any changes to the blueprints yet (and there are going to be quite a few), but I am happy to add them to this PR

Reply:
As discussed in a Slack thread, let's set aside the blueprints for now, we can do it in a follow-up ticket.